Frontend/ui-driver/src/Helpers/Utils.purs

```diff
@@ -411,4 +416,37 @@ onBoardingSubscriptionScreenCheck onBoardingSubscriptionViewCount isEnabled = is
                                                                               getValueToLocalNativeStore DRIVER_SUBSCRIBED == "false" && 
                                                                               even onBoardingSubscriptionViewCount && 
                                                                               onBoardingSubscriptionViewCount <5 && 
-                                                                              isOnFreeTrial FunctionCall
\ No newline at end of file
+                                                                              isOnFreeTrial FunctionCall
+
+getVehicleVariantImage :: String -> String
```

Comment:
@ashishsingh101 can we club these two functions : `getVehicleVariantImage` and `getVariantRideType` as the condition for case match is same for both.

Reply:
why to club both have diff. functionality, one is for image and other for string

Reply:
it can return an object.